lean.bib

```diff
@@ -373,6 +412,15 @@ @Misc{            Huisinga19
   tags          = {about-lean, lean4}
 }
 
+@Booklet{         Kudryashov2021,
+  author        = {Yury Kudryashov},
+  title         = {Formalizing Rotation Number and Its Properties in Lean},
+  howpublished  = {EasyChair Preprint no. 6168},
+  url           = {https://easychair.org/publications/preprint/38w1},
+  year          = {2021},
+  tags          = {about-mathlib, lean3}
```

Comment:
What's the criterion for the `about-mathlib` tag? Some of the other papers (e.g. Eric's) could have that tag as well, right?

Reply:
This was a copy-paste mistake, I fixed it in the next commit. This one is clearly `formalization`. Eric's is maybe debatable.